boefjes/boefjes/plugins/kat_abuseipdb/normalize.py

```diff
@@ -0,0 +1,51 @@
+import json
+from collections.abc import Iterable
+from typing import Any
+
+import structlog
+
+from octopoes.models import OOI, Reference
+from octopoes.models.ooi.findings import Finding, KATFindingType
+
+REPORT_CATEGORIES_PATH = "boefjes/plugins/kat_abuseipdb/abuseipdb_report_categories.json"
```

Comment:
I suggest using a dynamic path instead based on the current file location, because this could potentially break in the near future
```suggestion
REPORT_CATEGORIES_PATH = Path(__file__).parent / "abuseipdb_report_categories.json"
```

Reply:
Agreed, looks a lot better this way.

I took this piece of code from previous boefjes, would you like me to update those boefjes as well in this PR?

Reply:
Would be nice!